.github/workflows/test-docker-build.yml

```diff
@@ -1,42 +1,61 @@
-name: Test Docker images (scheduled)
+name: Test Docker images (on PR)
 
 on:
   pull_request:
```

Comment:
I am not sure if we really want to test this in each PR (all of its commits), or just when merged into `main`.

I know it's better to keep PR green. But for workflow run on each PR, we have to manually approve the run (for security run), right?

ping @glegendre01 again to see if the infra team prefers not to allow rununing on PRs whenever possible, or it doesn't really matter.

Reply:
I think as long as we don't push the docker image on docker hub it's all good, from the conversation I had with guillaume !